src/components/modals/login-modal/LoginContent.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Form, Input } from 'antd';
+import React from 'react';
+import './login-content.less';
+import Text from 'antd/es/typography/Text';
+
+interface Props {
+  switchToForgotPasswordPage: () => void;
+}
+
+const LoginContent: (props: Props) => JSX.Element = (props: Props) => {
```

Comment:
```suggestion
interface LoginContentProps {
  readonly switchToForgotPasswordPage: () => void;
}

const LoginContent: JSX.Element<LoginContentProps> = ({
  switchToForgotPasswordPage,
}) => {
```
A few suggestions here: 
1. Always name the props interface `{ComponentName}Props`
2. You can use `<LoginContentProps>` to specify the component's props
3. Destructuring props in the component makes things easier to read and access